Content.Server/Body/Respiratory/RespiratorComponent.cs

```diff
@@ -34,8 +36,11 @@ public class RespiratorComponent : Component
         private bool _isShivering;
         private bool _isSweating;
 
-        [DataField("damageType", required: true)]
-        private readonly string _damageType = default!;
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [DataField("suffocationDamageType")]
```

Comment:
why the change from `damageType` to `suffocationDamageType`?

Reply:
Currently it aready uses 'suffocationDamage' and 'suffocationDamageRecovery' for the damage numbers. So I changed it to be consistent. Given that respirator doesn't deal any other damage, I'll just remove 'suffocation' from all of them. Also now that it's called Respirator instead of Metabolism or whatever it was before, it should hopefully be obvious what the damage type is for.